huey_monitor/tqdm.py

```diff
@@ -40,6 +41,15 @@ def __init__(self,
         self.unit_divisor = unit_divisor
         self.parent_task_id = parent_task_id
 
+        if len(self.desc) > 64:
```

Comment:
Can we dynamically determine the length of the description field?

Reply:
Yes, i do this in the test:

`max_length = TaskModel._meta.get_field('desc').max_length`

but i think it's faster to use a static value...
If this value is not the right one, the test will fail.